N2V/N2V_GUI_OMERO_v2.py

```diff
@@ -222,19 +237,22 @@ def configure_N2V(self):
         try:
             epochs = int(self.train_epochs.get())
         except ValueError:
-            tk.messagebox.showwarning(title="Wrong entry", message="Please enter a number!")
+            tk.messagebox.showwarning(title="Wrong entry", message="Please enter a number for epochs!")
             train_steps=0
             epochs=0
 
+        try:
+            train_batch_size = int(self.train_batchsize.get())
+        except ValueError:
+            tk.messagebox.showwarning(title="Wrong entry", message="Please enter a number for the batch size!")
+            train_steps=0
+            train_batch_size=0
+
         shapes = []
         for i in range(len(self.imgs)):
             shapes.append(self.imgs[i].shape[1])
-        if self.three_D:

```

Comment:
Removing this part, might lead to OOM errors. 
Might be good to keep an eye on it, and maybe re-add a maximum batch-size for large or 3D images